static/js/entry/zendesk_widget.js

```diff
@@ -51,7 +47,7 @@ window.zEmbed ||
     o.close()
   })(
     "https://static.zdassets.com/ekr/snippet.js?key=0c7cfbbc-76c9-4083-869f-b76d03922056",
-    "odl.zendesk.com"
+    "mitx-micromasters.zendesk.com"
```

Comment:
Huh. Do you think this might have been the cause of our problem with the wrong theme loading?

Reply:
It was mainly because of that but along with that the custom code was also touching the colors that got removed.

Reply:
I had set up the trial account on Zendesk where I put up the web widget which made me troubleshoot that.